### PR TITLE
Es/plug translation

### PR DESCRIPTION
**Pull request by edwinallenz**
v1.1.0 Plug translation to spanish related #1043
cc @erickgnavar

### CODE REVIEW

**Comment by doomspork** [comment]
Howdy @edwinallenz!  Thank you again for your contributions.

It looks like you've got a lot of extra commits on your branch, could you rebase?

**Comment by edwinallenz** [comment]
will do and let you know @doomspork

File: es/lessons/specifics/plug.md [hunk 1]
```diff
@@ -59,6 +70,14 @@ end
 
 Guarda el archivo en `lib/example/hello_world_plug.ex`.
 
+La función `init/1` se utiliza para inicializar las opciones de nuestros Plugs. Esta es llamada por el árbol de supervisión, el cual se explica en la siguiente sección. De momento, está será una Lista vacía que es ignorada.
```
Comment (erickgnavar) [inline es/lessons/specifics/plug.md]: usar minúscula Lista -> **l**ista

**Comment by edwinallenz** [comment]
@erickgnavar Done.
cc @doomspork

**Comment by doomspork** [comment]
Awesome!  Thank you @edwinallenz and @erickgnavar

**Pull request merged**